### PR TITLE
Updates to CHANGELOG for 6.3

### PR DESCRIPTION
**Pull request by causten**
Changelog updates based on taking all the commits in 6.3 since 6.2 and bowling them down to human readable descriptions of the changes.  All PRs where previously shared in a xls with the team to categorize.  Please review for grammar

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,85 @@
...
+
+* Added initial code to run on Windows
+* Added support for gfx120x GPU 
+* Added support for FP8, and INT4
+* Added ONNX operators Bitwise And, Scan,SoftmaxCrossEntropyLoss, GridSample, NegativeLogLikelihoodLoss
+* Added Microsoft Contrib operators MatMulNBits, QuantizeLinear/DequantizeLinear, GroupQueryAttention
+* Added support for the Log2 internal operator
+* Added Split-K as an optional performance improvement
+* Added support for the GCC 14 compiler
+* Added the ablity to call hipBlasLt libaries using environment variable MIGRAPHX_ENABLE_HIPBLASLT_GEMM=1
```
Comment (ahsan-ca) [inline CHANGELOG.md:18]:
```suggestion
* Added the ablity to call hipBLASLt libraries using environment variable MIGRAPHX_ENABLE_HIPBLASLT_GEMM=1
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,85 @@
...
+* Added a flag to dump mlir modules to mxrs
+* Added MIGRAPHX_TRACE_BENCHMARKING=3 to print the MLIR program for improved debugability
+
+
+### Optimizations
+
+* Optimized the NHWC layout to improve performance of many convolution based models
+* Improved GPU utilization
+* Improved infrastructure code to enable better Kernel fusions with all supported data types
+* Optmized performance for the FP8 datatype
```
Comment (ahsan-ca) [inline CHANGELOG.md:39]:
```suggestion
* Optimized performance for the FP8 datatype 
```

**Review by ahsan-ca — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,85 @@
...
+* Added support for the Log2 internal operator
+* Added Split-K as an optional performance improvement
+* Added support for the GCC 14 compiler
+* Added the ablity to call hipBlasLt libaries using environment variable MIGRAPHX_ENABLE_HIPBLASLT_GEMM=1
+* Added scripts to validate ONNX models from the ONNX Model Zoo
+* Enabled the OneHot operator to accept a dynamic batch parameter 
+* Added a --mlir flag the migraphx-driver program to offload entire module to mlir
+* Added GPU Pooling Kernel
+* Added examples for RNNT, and ControlNet
+* Added support for Skip and Simplified LayerNormalization
```
Comment (TedThemistokleous) [inline CHANGELOG.md:24]: Skip Simplified LayerNormalization and Simplied LayerNormalization should be with Microsoft contrib operators

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,85 @@
...
+* Added support for FP8, and INT4
+* Added ONNX operators Bitwise And, Scan,SoftmaxCrossEntropyLoss, GridSample, NegativeLogLikelihoodLoss
+* Added Microsoft Contrib operators MatMulNBits, QuantizeLinear/DequantizeLinear, GroupQueryAttention
+* Added support for the Log2 internal operator
+* Added Split-K as an optional performance improvement
+* Added support for the GCC 14 compiler
+* Added the ablity to call hipBlasLt libaries using environment variable MIGRAPHX_ENABLE_HIPBLASLT_GEMM=1
+* Added scripts to validate ONNX models from the ONNX Model Zoo
+* Enabled the OneHot operator to accept a dynamic batch parameter 
+* Added a --mlir flag the migraphx-driver program to offload entire module to mlir
```
Comment (TedThemistokleous) [inline CHANGELOG.md:21]: remove the `a` before `--mlir`

Comment (ahsan-ca) [inline CHANGELOG.md:21]:
```suggestion
* Added --mlir flag to the migraphx-driver program to offload entire module to mlir
```

**Comment by causten** [comment]
> I'll create a separate PR later for putting this in the format used in the release notes, but it's good for now.

I merged your changes here already

**Pull request merged**